tests/functional/object/mcg/test_lifecycle_configuration.py

```diff
@@ -0,0 +1,365 @@
+from datetime import datetime, timedelta
+import json
+import logging
+from time import sleep
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    red_squad,
+    runs_on_provider,
+    mcg,
+    skipif_noobaa_external_pgsql,
+)
+from ocs_ci.framework.testlib import MCGTest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.bucket_utils import (
+    change_versions_creation_date_in_noobaa_db,
+    create_multipart_upload,
+    expire_multipart_upload,
+    get_obj_versions,
+    list_multipart_upload,
+    list_objects_from_bucket,
+    put_bucket_versioning_via_awscli,
+    s3_delete_object,
+    s3_list_object_versions,
+    upload_obj_versions,
+    upload_parts,
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources.mcg_lifecycle_policies import (
+    AbortIncompleteMultipartUploadRule,
+    ExpirationRule,
+    LifecyclePolicy,
+    NoncurrentVersionExpirationRule,
+)
+from ocs_ci.utility.utils import TimeoutSampler, exec_nb_db_query
+
+
+logger = logging.getLogger(__name__)
+
+PROP_SLEEP_TIME = 10
+TIMEOUT_SLEEP_DURATION = 30
+TIMEOUT_THRESHOLD = 600
+
+
+@mcg
+@red_squad
+@runs_on_provider
+@skipif_noobaa_external_pgsql
+class TestLifecycleConfiguration(MCGTest):
+    """
+    Tests suite for lifecycle configurations on MCG
+
+    """
+
+    @pytest.fixture(scope="class", autouse=True)
+    def reduce_expiration_interval(self, add_env_vars_to_noobaa_core_class):
+        """
+        Reduce the interval in which the lifecycle background worker is running
+
+        """
+        new_interval_in_miliseconds = 60 * 1000
+        add_env_vars_to_noobaa_core_class(
+            [
+                (constants.LIFECYCLE_INTERVAL_PARAM, new_interval_in_miliseconds),
+                (constants.LIFECYCLE_SCHED_MINUTES, 1),
+            ]
+        )
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-6541")
+    def test_abort_incomplete_multipart_upload(
+        self, mcg_obj, bucket_factory, awscli_pod, test_directory_setup
+    ):
+        """
```

Comment:
add args in docstring

Reply:
We typically don't include fixture args in docstrings for test functions, so I think it's best to keep it consistent with the rest of the project.